### PR TITLE
Rego azure iac

### PR DESCRIPTION
**Pull request by shahinshirvani**

### CODE REVIEW

**Review by farchide — CHANGES_REQUESTED** [review]

File: azure/iac/KeyVault.rego [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    azure_issue["enableSoftDelete"]
+}
+
+
+enableSoftDelete_metadata := {
+    "Policy Code": "PR-AZR-0108-ARM",
+    "Type": "IaC",
+    "Product": "AZR",
+    "Language": "ARM template",
+    "Policy Title": "ENSURE THE KEY VAULT IS RECOVERABLE - ENABLE 'SOFT DELETE' SETTING FOR A KEY VAULT",
```
[the remark this answers is not in the document]
Comment (shahinshirvani) [reply azure/iac/KeyVault.rego]: I fixed that

File: azure/iac/LogProfile.rego [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+
+# PR-AZR-0120-ARM
+
+default locations = null
+
+azure_issue["locations"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.insights/logprofiles"
+    count(resource.properties.locations) < 65
```
[the remark this answers is not in the document]
Comment (shahinshirvani) [reply azure/iac/LogProfile.rego:64]: I fixed that

File: azure/iac/KeyVault.rego [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    azure_issue["KeyVault"]
+}
+
+
+KeyVault_metadata := {
+    "Policy Code": "PR-AZR-0107-ARM",
+    "Type": "IaC",
+    "Product": "AZR",
+    "Language": "ARM template",
+    "Policy Title": "ENSURE THAT KEYVAULT IS IN USE",
```
[the remark this answers is not in the document]
Comment (shahinshirvani) [reply azure/iac/KeyVault.rego]: I fixed that

File: azure/iac/KeyVault.rego [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+# https://docs.microsoft.com/en-us/azure/templates/microsoft.keyvault/vaults
+
+# PR-AZR-0107-ARM
+
+default KeyVault = null
+
+azure_attribute_absence["KeyVault"] {
+    resource := input.resources[_]
+    lower(resource.type) == "microsoft.keyvault/vaults"
+    accessPolicie := resource.properties.accessPolicies[_]
```
Comment (rezoan) [inline azure/iac/KeyVault.rego:12]: accessPolicie should be accessPolicy

**Comment by rezoan** [comment]
@farchide @shahinshirvani any reason to close this PR without fixing all the issues mentioned in the review?

**Pull request closed without merging**